Source/CesiumRuntime/Private/Cesium3DTileset.cpp

```diff
@@ -1817,17 +1877,34 @@ void ACesium3DTileset::Tick(float DeltaTime) {
   }
 
   const Cesium3DTilesSelection::ViewUpdateResult& result =
-      this->_captureMovieMode ? this->_pTileset->updateViewOffline(frustums)
-                              : this->_pTileset->updateView(frustums);
+      this->_captureMovieMode
+          ? this->_pTileset->updateViewOffline(frustums)
+          : this->_pTileset->updateView(frustums, DeltaTime);
   updateLastViewUpdateResultState(result);
   this->UpdateLoadStatus();
 
   removeVisibleTilesFromList(
       this->_tilesToNoLongerRenderNextFrame,
-      result.tilesToRenderThisFrame);
-  hideTilesToNoLongerRender(this->_tilesToNoLongerRenderNextFrame);
-  this->_tilesToNoLongerRenderNextFrame = result.tilesToNoLongerRenderThisFrame;
-  showTilesToRender(result.tilesToRenderThisFrame);
+      result.tilesSelectedThisFrame);
+  removeVisibleTilesFromList(
+      this->_tilesToHideNextFrame,
+      result.tilesSelectedThisFrame);
```

Comment:
I maybe wrong here but `_tilesToNoLongerSelectedNextFrame` should consists of `_tilesToHideNextFrame` right? If so, is the removal of `_tilesToHideNextFrame` not needed right?